modules/filters/mod_deflate.c

```diff
@@ -1298,6 +1298,7 @@ static apr_status_t deflate_in_filter(ap_filter_t *f,
         {
             const char *data;
             apr_size_t len;
+            apr_bucket *tmp_b;
```

Comment:
Can't we move this to line 1354? The block starting there seems to be the only one that needs it.